packages/transform-metering/src/meter.js

```diff
@@ -7,8 +6,8 @@ const { ceil } = Math;
 const ObjectConstructor = Object;
 
 // eslint-disable-next-line no-bitwise
-const bigIntWord = typeof BigInt !== 'undefined' && BigInt(2) ** BigInt(64);
-const bigIntZero = bigIntWord && BigInt(0);
+const bigIntWord = typeof BigInt !== 'undefined' && 2n ** 64n;
+const bigIntZero = bigIntWord && 0n;
```

Comment:
This was overly clever to be able to cope with a non-BigInt world.  We can simplify (and remove the earlier `// eslint-disable-next-line no-bitwise`):

```suggestion
const bigIntWord = 2n ** 64n;
const bigIntZero = 0n;
```